apps/studio/components/ui/GlobalErrorBoundaryState.tsx

```diff
@@ -132,14 +133,14 @@ export const GlobalErrorBoundaryState = ({ error, resetErrorBoundary }: Fallback
         )}
 
         <div className="w-full sm:w-1/2 mx-auto grid grid-cols-2 gap-2">
-          <Button asChild type="default" icon={<ExternalLink />}>
-            <Link
-              href={`/support/new?category=dashboard_bug&subject=Client%20side%20exception%20occurred%20on%20dashboard&message=${encodeURI(urlMessage)}`}
-              target="_blank"
-            >
-              Contact support
-            </Link>
-          </Button>
+          <ContactSupportButton
```

Comment:
for this case specifically, we might be sending duplicate sentry events
since we already send an event to sentry when the user lands on the error boundary [here](https://github.com/supabase/supabase/blob/master/apps/studio/pages/_app.tsx#L84)